CRVResponse/src/MakeCrvDigis.cc

```diff
@@ -8,13 +8,12 @@ void MakeCrvDigis::SetWaveform(const std::vector<double> &waveform, double ADCco
   _ADCs.clear();
```

Comment:
```suggestion
  _ADCs.clear();
  _ADCs.reserve(waveform.size();
```

Reply:
I will add a reserve().

Reply:
As you revisit code, please always look for cases where you know the size of the vector and use reserve.

Reply:
Ping for this comment.